### DISCUSSION TITLE
Where is the _underscore_?

### DISCUSSION THREAD

**Discussion by rfalanga** [skills/.github#77] [Q&A]
I'm sorry, I've made a mistake, somewhere, but don't know where. In the GitHub skill for continuous integration, under the _Activity: Fix the test_ on step 2 the instructions say, "Update the code in the `ci` branch to get the test to pass. You need to look something like this: \_underscore\_". I went to the Code tab. I made sure I was on the `ci` branch, but I cannot find any file with \_underscore\_ in it. I'm sorry, would you please direct me to where I should look?

**Comment by sinsukehlab** [comment]
As far as I know, you haven't made a mistake. Keep it up!
The course maintainers have made ci.yml raise an error on purpose.
There is a comment `<!--[comment]-->` for the learners in README.md.
You should fix the line below this comment.
You can check the error message of ci.yml to know what and how should be changed.

**Comment by markpatterson27** [comment]
There's a typo in the instructions that makes the activity very unclear.

In the Actions log, you will find something like this:
![image](https://user-images.githubusercontent.com/35724907/197017178-c54363f7-15a1-4376-82c0-d9221a9d8f23.png)

It tells you exactly what the error is, and crucially, where it it: README.md file. `115:1-115:63` means, line 115 of README.md, spanning between 1st and 63rd characters

There's a [PR on the template](https://github.com/skills/continuous-integration/pull/13) with suggested improvements to make the activity clearer. If you have any thoughts on that you could leave a comment there.

**Comment by rfalanga** [comment]
Thank you very much, @markpatterson27. I'm not familiar enough with Markdown to have caught that.
